packages/utils/src/db/db.ts

```diff
@@ -981,18 +1009,18 @@ export class HoprDB {
     const result: PublicKey[] = []
     let SIZE_PUBKEY_COMPRESSED = PublicKey.size_compressed()
     let SIZE_PUBKEY_UNCOMPRESSED = PublicKey.size_uncompressed()
-    for (let offset = 0; offset < arr.length;) {
-      switch (arr[offset]) {
-        case 2:
-        case 3:
+    for (let offset = 0; offset < arr.length; ) {
```

Comment:
This is unnecessary though, this code will be removed once the e2e tests pass. The `PublicKey` uses the Rust serialization now. Everything inside the `HoprDb` class will disappear.

Reply:
True just to make sure it's documented